r/R/anime.R

```diff
@@ -81,6 +114,8 @@ print.anime <- function(x, ...) {
 
 #' Interpolate extensive variables
 #'
+#' @param x An `anime` object
+#' @param ... Variables to interpolate
```

Comment:
Can we say here specifically numeric variables

Reply:
Done